src/components/InputTodo.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-alert */
+/* eslint-disable react/button-has-type */
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+
+const InputTodo = (props) => {
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (inputText.title.trim()) {
+      // eslint-disable-next-line react/destructuring-assignment, react/prop-types
```

Comment:
- [OPTIONAL] Linters are tools that perform static analysis on software to detect and report adherence to a set of coding best practices. Instead of disabling eslint, it would be preferable to resolve the issue.
-  Check this link on how to handle [react/prop-types](https://medium.com/@leonardobrunolima/react-tips-validating-props-with-proptypes-24c64d58f4c0)
- Check this link on how to handle [react/destructuring-assignment](https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0